### PR TITLE
Adds open graph to docs

### LINKED ISSUE #200: Add Open Graph Info to Docs HTML
The current documentation does not contain any open graph information. This means that if someone shares a link to the docs via social media they will not get a nice image with the share. Open graph information should be added for better presentation during sharing.

For reference, here is the relevant code in Featuretools that adds this information:
https://github.com/alteryx/featuretools/blob/65025377fd1012d5ebd241602da79cd1926679c3/docs/source/templates/layout.html#L12-L25

The attached image can be used for the open graph image.
OpenSource_OpenGraph_1200x630px-compose

### LINKED ISSUE #205: Update links to use alteryx org
There are several links in the documentation that point to Featurelabs/compose instead of alteryx/compose and should be updated.

### PR DESCRIPTION
**Pull request by jeff-hernandez**
Closes #200 by adding open graph information to the documentation. Also closes #205 by updating the repository links.

### CODE REVIEW

File: docs/source/_templates/layout.html [hunk 1]
```diff
@@ -12,6 +12,21 @@
   }();
 </script>
 
+{% set image = 'https://user-images.githubusercontent.com/4307001/105420214-98052d00-5c05-11eb-86cc-1d1722099472.png' %}
```
Comment (rwedge) [inline docs/source/_templates/layout.html:15]: How is this being hosted?

Comment (jeff-hernandez) [reply docs/source/_templates/layout.html:15]: It's currently referencing the attached image [here](https://github.com/alteryx/compose/issues/200#issue-791539766). Do we want to save and reference it from somewhere else?

Comment (rwedge) [reply docs/source/_templates/layout.html:15]: let's host it on s3, similar to how we do for fearuretools

**Review by rwedge — APPROVED** [review]
Looks good!

**Pull request merged**